lib/process-manager.ts

```diff
@@ -22,6 +22,32 @@ const ROOT_DIR = path.resolve(__dirname, '..');
 export const processManagers: ProcessManager[] = [];
 export const disabled = false;
 
+export function exec(args: string, execOptions?: child_process.ExecOptions): Promise<string>;
+export function exec(args: [string, ...string[]], execOptions?: child_process. ExecFileOptions): Promise<string>;
```

Comment:
```suggestion
export function exec(args: [string, ...string[]], execOptions?: child_process.ExecFileOptions): Promise<string>;
```

Am surprised this is not caught by linter.